netutils/password.py

```diff
@@ -103,12 +106,14 @@ def compare_type5(unencrypted_password, encrypted_password, return_original=Fals
     salt = get_hash_salt(encrypted_password)
     if encrypt_type5(unencrypted_password, salt) == encrypted_password:
         if return_original is True:
-            return encrypted_password
+            return encrypted_password  # type: ignore
```

Comment:
Return value of the function needs to be `Union[str, bool]`, then we can get rid of this (but need to probably perform explicit handling of that return type in lots of places).